app/services/foreman_seeder.rb

```diff
@@ -2,6 +2,7 @@
 
 class ForemanSeeder
   FOREMAN_INTERNAL_KEY = 'database_seed'.freeze
+  ADVISORY_LOCK = 0x7b5b5e3c0549a95c  # random Int64, should probably saved centrally
```

Comment:
Actually I like that the lock is here and not saved centrally :) since it shouldn't be used anywhere else